WinUIGallery/Controls/DesignGuidance/ColorTile.xaml

```diff
@@ -0,0 +1,44 @@
+<!-- Copyright (c) Microsoft Corporation and Contributors. -->
+<!-- Licensed under the MIT License. -->
+
+<UserControl
+    x:Class="WinUIGallery.DesktopWap.Controls.DesignGuidance.ColorTile"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:WinUIGallery.DesktopWap.Controls.DesignGuidance"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid Padding="10,12" RowSpacing="6"
+        Background="{x:Bind Background, Mode=OneWay}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*" MinHeight="30"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*"/>
+            <ColumnDefinition Width="Auto"/>
+        </Grid.ColumnDefinitions>
+
+        <TextBlock Foreground="{x:Bind Foreground, Mode=OneWay}" TextWrapping="WrapWholeWords"
+            Text="{x:Bind ColorName, Mode=OneWay}" Style="{ThemeResource BodyStrongTextBlockStyle}"/>
+        <Button Grid.Column="1" AutomationProperties.Name="Copy brush name"
+                Background="Transparent" BorderBrush="Transparent"
+                Foreground="{x:Bind Foreground, Mode=OneWay}"
+                ToolTipService.ToolTip="Copy brush name" VerticalAlignment="Top"
+                HorizontalAlignment="Right" Click="CopyBrushNameButton_Click" Padding="4">
+            <FontIcon Glyph="&#xE8C8;" FontSize="16"/>
+        </Button>
+
+        <TextBlock Text="{x:Bind ColorExplanation, Mode=OneWay}" Grid.Row="1"
```

Comment:
This needs to CaptionTextBlockStyle as per design spec.. will save some space as well :)!